README.md

```diff
@@ -1,4 +1,331 @@
+| `ocm-backplane project <project_name>`                                      | Manipulate the Kubeconfig and set the namespace of the current context to `project_name` |
+| `ocm-backplane script describe <script> [flags]`                            | Describe the given backplane script                                                      |
+| `ocm-backplane script list [flags]`                                         | List available backplane scripts                                                         |
+| `ocm-backplane status`                                                      | Print essential cluster info                                                             |
+| `ocm-backplane managedJob create <script> [flags]`                          | Create a backplane managed job resource                                                  |
+| `ocm-backplane managedJob get <job_name> [flags]`                           | Retrieve a backplane managed job resource                                                |
+| `ocm-backplane managedJob list [flags]`                                     | Retrieve a list of backplane managed job resources                                       |
+| `ocm-backplane managedJob logs <job_name> [flags]`                          | Retrieve logs of the specified managed job resource                                      |
+| `ocm-backplane managedJob delete <job_name> [flags]`                        | Delete the specified managed job resource                                                |
+| `ocm-backplane testJob create <script> [flags]`                             | Create a backplane test job on a non-production cluster for testing                      |
+| `ocm-backplane testJob get <job_name> [flags]`                              | Retrieve a backplane test job resource                                                   |
+| `ocm-backplane testJob list [flags]`                                        | Retrieve a list of backplane test job resources                                          |
+| `ocm-backplane testJob logs <job_name> [flags]`                             | Retrieve logs of the specified test job resource                                         |
+| `ocm-backplane upgrade`                                                     | Upgrade backplane-cli to the latest version                                              |
+| `ocm-backplane version`                                                     | Display the installed backplane-cli version                                              |
+
+### Example
+
+In this example, we will login to a cluster with id `123456abcdef` in production environment, and we have the OCM client environment setup [like this](https://github.com/openshift/ops-sop/blob/master/v4/knowledge_base/ocm.md#prod-and-staging-at-the-same-time).
```

Comment:
Let's use https://github.com/openshift-online/ocm-cli#log-in in stead of the private sop repo.